### DISCUSSION TITLE
Auto CQ Reply

### DISCUSSION THREAD

**Discussion by st599** [danleetw/FT8TW#1] [Ideas]
Would it be possible to be able to have a choice available for CQ replies?

e.g. Furthest, best signal level etc.

**Comment by danleetw** [comment]
If I recently completed the PskReporter and QRZ upload features, I do want to try selecting the nearest, the furthest, the strongest signal, or the first callsign that responded to me for the next QSO. I also noticed that the original QSO logic seems to stop automatic responses under certain conditions and gets stuck repeatedly responding to the other party's grid.

**Comment by danleetw** [comment]
> Would it be possible to be able to have a choice available for CQ replies?
> 
> e.g. Furthest, best signal level etc.

Just finished.😄

**Comment by danleetw** [comment]
Release Notes
New Feature: CQ Sequence Options

We have introduced multiple CQ sequence prioritization schemes to better match different operating preferences. Regardless of the selected scheme, **stations calling your callsign are always given the highest priority**. The prioritization options include:

0:ASAP (Default)
Focuses on the fastest possible response based on SNR.

1:
Distance (Dist)
Prioritizes stations based on geographical distance.

2:ICD
Considers stations within new ITU Zones, CQ Zones, or DXCC entities.

3:ITU Zone Focus
Emphasizes ITU Zones over CQ Zones and DXCC.

4:CQ Zone Focus
Prioritizes CQ Zones above ITU Zones and DXCC.

5:DXCC Zone Focus
Gives higher priority to stations from new DXCC entities.

**Comment by danleetw** [comment]
> Would it be possible to be able to have a choice available for CQ replies?
> 
> e.g. Furthest, best signal level etc.


Completed。

**Comment by ZB2ham** [comment]
Autoreplying to CQ is fine IF YOU WANT IT
Autocompleting autosequencing a QSO is fine IF YOU WANT IT
How can I switch off these automatic functions, reverting to fully manual operation?
How can I switch off having every CQ call visible all the time?
